app/lib/efile/ny/it201.rb

```diff
@@ -75,7 +75,7 @@ def calculate
         @it213.calculate
         set_line(:IT201_LINE_63, -> { @lines[:IT213_LINE_14].value })
         @it215.calculate
-        set_line(:IT201_LINE_65, -> { @lines[:IT215_LINE_16].value })
+        set_line(:IT201_LINE_65, :calculate_line_65)
```

Comment:
I see that there is a calculate method for it215, which also conditionally sets line 1 instead of exiting if line 1 is false. do you think that is worth updating? it would replace the need for :calculate_line_65

Reply:
it works as is, just something to consider

Reply:
With the `calculate_line_65` method we get some level of accountability in the explain calculations section - which is useful
![image](https://github.com/codeforamerica/vita-min/assets/17094895/1091fa76-8ba7-4f12-8875-240c774dfb65)

Reply:
nice, that's convenient. seems good the way it is then